apps/api/src/main.ts

```diff
@@ -19,6 +20,16 @@ async function bootstrap(): Promise<void> {
   const app = await NestFactory.create(AppModule, {
     logger: loggerConfig,
   });
+  const config = new DocumentBuilder()
+    .setTitle('OsmoX')
+    .setDescription(
+      'Centralized multi-channel notification management component for streamlined communication across email, SMS, WhatsApp, and push notifications.',
+    )
+    .setVersion('2.0.0')
```

Comment:
Can we get Title, description and version from package.json file instead of hard-coding it here.

Reply:
Updated the code to import Title, description and version from package.json file and use instead of hard coded values.